auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe this? https://github.com/odoo/odoo/issues/6287#issuecomment-240459221 YAML is not listed